src/components/Calendars/NutritionistCalendar.tsx

```diff
@@ -0,0 +1,128 @@
+        setCurrentMonth(format(firstDayNextMonth, 'MMM-yyyy'));
+    }
+
+    let selectedDayMeetings = appointments.filter((consultation) => isSameDay(parseISO(consultation.attributes.date), selectedDay));
+
+    return (
+        <div className="pt-16">
+            <div className="max-w-md px-4 mx-auto sm:px-7 md:max-w-4xl md:px-6">
+                <div className="md:grid md:grid-cols-2 md:divide-x md:divide-gray-200">
+                    <div className="md:pr-14">
+                        <div className="flex items-center">
+                            <h2 className="flex-auto font-semibold text-gray-900">{format(firstDayCurrentMonth, 'MMMM yyyy', { locale: pt })}</h2>
+                            <button type="button" onClick={previousMonth} className="-my-1.5 flex flex-none items-center justify-center p-1.5 text-gray-400 hover:text-gray-500">
+                                <span className="sr-only">Mês Passado</span>
+                                <ChevronLeftIcon className="w-5 h-5" aria-hidden="true" />
+                            </button>
+                            <button onClick={nextMonth} type="button" className="-my-1.5 -mr-1.5 ml-2 flex flex-none items-center justify-center p-1.5 text-gray-400 hover:text-gray-500">
+                                <span className="sr-only">Mês Seguinte</span>
+                                <ChevronRightIcon className="w-5 h-5" aria-hidden="true" />
+                            </button>
+                        </div>
+                        <div className="grid grid-cols-7 mt-10 text-xs leading-6 text-center text-gray-500">
+                            <div>S</div>
+                            <div>M</div>
+                            <div>T</div>
+                            <div>W</div>
+                            <div>T</div>
+                            <div>F</div>
+                            <div>S</div>
+                        </div>
+                        <div className="grid grid-cols-7 mt-2 text-sm">
+                            {days.map((day: Date, dayIdx: number) => (
+                                <div key={day.toString()} className={`${dayIdx === 0 && colStartClasses[getDay(day)]} py-1.5`}>
+                                    <button type="button" onClick={() => setSelectedDay(day)} className={`${isEqual(day, selectedDay) && 'text-white'} ${!isEqual(day, selectedDay) && isToday(day) && 'text-red-500'} ${!isEqual(day, selectedDay) && !isToday(day) && isSameMonth(day, firstDayCurrentMonth) && 'text-gray-900'} ${!isEqual(day, selectedDay) && !isToday(day) && !isSameMonth(day, firstDayCurrentMonth) && 'text-gray-400'} ${isEqual(day, selectedDay) && isToday(day) && 'bg-red-500'} ${isEqual(day, selectedDay) && !isToday(day) && 'bg-gray-900'} ${!isEqual(day, selectedDay) && 'hover:bg-gray-200'} ${(isEqual(day, selectedDay) || isToday(day)) && 'font-semibold'} mx-auto flex h-8 w-8 items-center justify-center rounded-full`}>
```

Comment:
Wtf is happening on this button? XD

Reply:
Please can we abstract this to a function, give a name for be understandable ? <3